.github/workflows/codecov.yml

```diff
@@ -30,4 +30,7 @@ jobs:
         with:
           file: ./coverage.out
           fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
\ No newline at end of file
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+      - name: Validate codecov config
```

Comment:
[SUGGESTION] It would be better if you did it before `codecov/codecov-action@v4`

Reply:
Moreover, you can do it once.